### DISCUSSION TITLE
Does it work with Microfrontends created using Module-Federation?

### DISCUSSION THREAD

**Discussion by puthranankit** [microsoft/redux-micro-frontend#41] [Q&A]
Does it work with Microfrontends created using Module-Federation?

**Comment by tmonikat** [comment]
Yes, I have worked on React - Module Federation and Redux Micro frontend. Works like charm.

**Reply by puthranankit** [reply]
okay , Even I've used it with module federation , But I  was seeing one issue where I was able to dispatch actions which were not registered under global action . Wondering you guys might've observed the same issue as well. I referred the code provided in their sample

**Reply by AlfeeAeman** [reply]
@tmonikat i am facing issue with functional component 
Can you please help

**Reply by tmonikat** [reply]
> okay , Even I've used it with module federation , But I was seeing one issue where I was able to dispatch actions which were not registered under global action . Wondering you guys might've observed the same issue as well. I referred the code provided in their sample

We didn’t use Global action feature as it wasn’t required in our system. For any common actions which is used between the microfrontends, we put them in common util library.
I remember seeing an issue reported on the same issue as yours. Can you add ur issue here
https://github.com/microsoft/redux-micro-frontend/issues/25

**Reply by tmonikat** [reply]
> @tmonikat i am facing issue with functional component
> Can you please help

What is the issue?

**Reply by AlfeeAeman** [reply]
![image](https://user-images.githubusercontent.com/94458168/173504394-99b03f75-88dc-4c68-a71d-4b52f58e8b82.png)
let globalStore = GlobalStore.Get();
